File: cf-816/a.cpp
#include <bits/stdc++.h>

using namespace std;

int main(int argc, char const *argv[])
{
	int tt;
	cin >> tt;

	while(tt--){
		int n, k;
		cin >> n >> k;

		int mn = min(n, k);

		int mx = max(n, k);

		int ans = (mx + mn) + mn - 2;

		cout << ans << '\n';
	}
	return 0;
}

File: cf-816/a.py
tt = int(input())

for i in range(tt):
	n, m = map(int, input().split())

	mn = min(n, m)
	mx = max(n, m)

	ans = mn + mn + mx - 2

	if(n == 1 and m == 1):
		ans = 0

	print(ans)